Tests/conf.json

```diff
@@ -3191,10 +3195,11 @@
         "Lastline v2",
         "McAfee DXL",
         "GCP Whitelist Feed",
-        "Cortex Data Lake"
+        "Cortex Data Lake",
+        "AWS - Security Hub"
```

Comment:
are we ok with updating findings in parallel?

Reply:
Yes, I'm updating only one finding in the playbook and it's a random one, so the worst case is that it will verify a finding was updated while it was updated by another job, It'll pass anyways since the update is the same